defaults/severities.yaml

```diff
@@ -98,3 +98,6 @@ ScyllaServerStatusEvent: NORMAL
 BootstrapEvent: NORMAL
 RepairEvent: NORMAL
 JMXServiceEvent: NORMAL
+InstanceStatusEvent.STARTUP: WARNING
```

Comment:
Need to remove `DatabaseLogEvent.POWER_OFF: WARNING` line in this file

Reply:
removed, thanks